### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by ChristopherHammond13**
Okay, so this is a big 'un...
Essentially, when this was first created I can only assume that instead of cloning from lord/slate upstream, the files were copied and pasted in? I've now updated our Slate setup with all the commits upstream from @lord. This is awesome, but it means we now need to merge 462 more commits than what I **actually** did into master. Time for a LMAO? Probably. lmao.

Anyway, aside from upgrading us to Slate 1.5.0 (the latest, uploaded yesterday, with support for multiple of the same language in the Markdown), I have created tabs with examples of including the `uclapi-roombookings-version` header that we'll be checking for after going live. Once we're happy we should tag this release as `1.0`.

I also added some explanation about the versioning system and why it'll be in place, but I'm not sure if this is in the right place or worded how you guys think would be best so do feel free to read it.

Ignore my [WIP DO NOT MERGE] commits: they can be merged in now that I have fixed the masses of bugs. I had to manually copy over bits of Slate because it was a complete nightmare to merge things in. It was far from clean... I think it might be worth squashing those three commits together but I'm not completely confident doing that yet, and I'd rather keep the full history in case I screwed stuff up.

Here are some screenies...
image

image

### CODE REVIEW

**Comment by xoneco** [comment]
Holy shit. that's a lot of commits

**Comment by ChristopherHammond13** [comment]
@xoneco not wrong! 😆

**Comment by lord** [comment]
My bad — I like using lots of commits so my github grid looks more green :)

**Comment by ChristopherHammond13** [comment]
@lord Completely forgot that tagging you would actually bring you onto this thread, so apologies! Thanks so much for Slate though; it's amazing :D

**Comment by ChristopherHammond13** [comment]
Turns out I hadn't resolved every merge conflict, so that's done now. I've also updated the Gemfile.lock so that we're on the latest versions of the Ruby Gems required to build Slate.

**Comment by xoneco** [comment]
@ChristopherHammond13 can we just use the header everywhere in the examples and not have two different versions? I think that's probably more confusing than useful.

**Comment by xoneco** [comment]
I thought it was too different language versions lol. Like python2 or python3

**Comment by xoneco** [comment]
image
 this seems a bit messed up

**Comment by xoneco** [comment]
image

The header section is slightly too long imho. We could make a separate section for it (`#`) which would also appear in the sidebar as well.

Also the last paragraph should probably be shorter? Most folks probably haven't used the enghub API

**Comment by xoneco** [comment]
I do like the versioning in the examples! 👌 And that the code theme is now monokai hehe
Thoughts around what version we should actually call this? as in `1` vs `1.0`

**Comment by ChristopherHammond13** [comment]
@xoneco I've rewritten the version stuff now (and made use of the `<aside>` tag, hehe), so feel free to CR.

**Comment by xoneco** [comment]
Ahh I like the red stuff 😂 !!
Nice!

File: source/index.html.md [hunk 1]
```diff
@@ -363,5 +420,8 @@ Error                 | Description
 `No siteid supplied`  | Gets returned when you don't supply a `siteid`.
 
 # Contributing
-These docs are open sourced at [https://github.com/uclapi/apiDocs](https://github.com/uclapi/apiDocs).  
-Any and all contributions are welcome! If you spot a type or error, fix it :)
+These docs are open sourced at [https://github.com/uclapi/apiDocs](https://github.com/uclapi/apiDocs).
+
+The full API is open sourced at [https://github.com/uclapi/uclapi](https://github.com/uclapi/uclapi).
```
Comment (xoneco) [inline source/index.html.md]: Not yet. If you want to merge this now I'd suggest taking this line out. Unless @jermenkoo has already completed the review and is happy to opensource uclapi/uclapi?

File: source/index.html.md [hunk 2]
```diff
@@ -363,5 +420,8 @@ Error                 | Description
...
+
+Any and all contributions are welcome! If you spot a typo or error, feel free to fix it and submit a pull request :)
```
Comment (xoneco) [inline source/index.html.md]: Nooooooo don't fixe the `type` typo

File: source/index.html.md (hunk 1 above)
Comment (ChristopherHammond13) [inline source/index.html.md]: I assumed we would open source the entire API once we go live on Tuesday?

File: source/index.html.md (hunk 2 above)
Comment (ChristopherHammond13) [inline source/index.html.md]: but..but.... **grumbles**

_fine_ :P

File: source/index.html.md (hunk 1 above)
Comment (xoneco) [inline source/index.html.md]: Yes definitely. I'm just being overly adamant that docs are in sync with the real world. But you're right. This is not that big a thing. Feel free to leave it in even now.

Comment (ChristopherHammond13) [inline source/index.html.md]: Ah fair enough; it's your shout anyway

**Comment by xoneco** [comment]
image

Is there any way we can make this more concise? It seems like such a wall of text

**Pull request merged**